.github/workflows/semgrep-analysis.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow file requires a free account on Semgrep.dev to
+# manage rules, file ignores, notifications, and more.
+#
+# See https://semgrep.dev/docs
+
+name: Semgrep
+
+on:
+  push:
+    branches:
+      - feature/annotations
```

Comment:
Can we do wildcard `feature/*` here? Also why not add `Dev` and `Master` as well?

Reply:
currently only targeting push to `feature/annotations`, all PRs and on schedule.

When `feature/annotations` be merged, it will updated to `dev`, `master` like other actions.